next-env.d.ts

```diff
@@ -0,0 +1,3 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
+/// <reference types="next/image-types/global" />
```

Comment:
This was added by Next.js for us. These "triple slash" type references bring in those types for all our files.

https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html